db/migrate/20160107161130_thumbs_up_migration.rb

```diff
@@ -0,0 +1,25 @@
+class ThumbsUpMigration < ActiveRecord::Migration
+  def self.up
+    create_table :votes, :force => true do |t|
+
+      t.boolean    :vote,     :default => false,    :null => false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.